### PR TITLE
docs: howto create patches

### PR DESCRIPTION
**Pull request by diafour**
- Commands to create patches for kubevirt or cdi

## Description

Add an internal page with commands to create patches for kubevirt or cdi.

## Why do we need it, and what problem does it solve?

It's easy to forget these infrequently executed commands.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: docs/internal/patches.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Howto create patches for kubectl and cdi
```
Comment (yaroslavborbat) [inline docs/internal/patches.md:1]:
```suggestion
### Howto create patches for kubevirt and cdi
```
kubectl?

File: docs/internal/patches.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Checkout kubevirt (or cdi) along with virtualization-controller. Use `$version` from werf.inc.yaml:
+```shell
+$ pwd
+/home/user/virtualization-controller
+$ cd ..
+$ git checkout github.com/kubevirt/kubevirt kubevirt
```
Comment (yaroslavborbat) [inline docs/internal/patches.md:8]: It seems strange to me.

**Review by yaroslavborbat — COMMENTED** [review]
We already have similar [docs](https://github.com/deckhouse/virtualization/blob/main/docs/internal/cdi.md).

**Comment by diafour** [comment]
I transform this PR to a few changes to existing docs.

**Review by yaroslavborbat — APPROVED** [review]

**Pull request merged**